test/schemas/operationSchemas.test.ts

```diff
@@ -352,6 +353,44 @@ describe('A decrement operation schema', () => {
     });
 });
 
+describe('A remove operation schema', () => {
+    test.each([
+        [1],
+        [1.5],
+        ['foo'],
+        [true],
+        [simpleArray],
+        [simpleMap],
+        [complexMap],
+    ])('should remove value %s', (value: any) => {
+        function validate(): void {
+            removeOperation.validate({path: 'foo', value: value});
+        }
+
+        expect(validate).not.toThrow(Error);
+    });
+
+    test.each([
+        ['foo.'],
+        [''],
+    ])('should fail if path is invalid', (path: string) => {
+        function validate(): void {
+            removeOperation.validate({path: path, value: 'bar'});
```

Comment:
To keep all tests standardized, please add the empty case on others as well, please.